### PR TITLE
Makes cloaks into Exosuits as god(I) intended.v2

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
I love cloaks, but they were meant to be exosuit items. This time armor values are fixed so they are a good cosmetic choice, without sacrificing armor. Shout out to coderbus and goof for help and feedback. 

:cl: 
tweak: Cloaks are now worn in your exosuit slot! 
/:cl:

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
oh my FUCKING GOD

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,73 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"
```
Comment (RemieRichards) [inline tgstation.dme]: dont commit ```FILE_DIR```

Dreammaker --> Help --> Preferences for tgstation --> Untick "automatically define FILE_DIR"

then recompile and recommit.

**Comment by GunHog** [comment]
I feel these are better off as jumpsuit accessories, maybe. They are rarely worn because they result in the inability to wear preferred items. Currently, that is a backpack/satchel. Making the cloaks into exosuit items will make them an acceptable style choice for some heads (RD, CMO, HoP) who do not always need to use their other exosuit slot items to do their job. For the Captain, HoS, and CE, these are still terrible choices.

If you made them jumpsuit items, then they can be just as much as a valid style choice as an alt jumpsuit is.

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -14,39 +14,39 @@
...
+/obj/item/clothing/suit/cloak/rd
 	name = "research director's cloak."
 	desc = "Worn by Sciencia, thaumaturges and researchers of the universe. It's slightly shielded from contaminants."
 	icon_state = "rdcloak"
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 10, rad = 0)
 
-/obj/item/clothing/cloak/cap
+/obj/item/clothing/suit/cloak/cap
 	name = "captain's cloak"
 	desc = "Worm by the commander of Space Station 13."
```
Comment (Shihrt) [inline code/modules/clothing/suits/cloaks.dm]: Typo "Worm by"

**Comment by TechnoAlchemisto** [comment]
@GunHog HoS cloak and Captain cloak had their armor values buffed. HoS cloak is equal to the greatcoat, captain cloak is slightly less than the carapace because it covers more than just the chest.

**Comment by SweaterKittens** [comment]
This is stupid, but I would really hate to not be able to wear my cloak/hardsuit as CE. I know it's a nitpick and certain jobs would really benefit from this change, but frankly I'd be disappointed.

**Comment by Miauw62** [comment]
cant you have them in like both slots

**Comment by GunHog** [comment]
Or a *jumpsuit accessory*...

**Comment by Cheridan** [comment]
wearing a backpack over a cloak is dumb

**Comment by TechnoAlchemisto** [comment]
@Cheridan I think you're confusing cloaks and capes. Cloaks have always been something you wore as an outer layer of clothing and bags have often gone over them historically as well as in other games.

@GunHog
Once again, that might make sense if they were a cape, however cloaks are not capes and are not things just tacked on to what you're wearing for fashion. 

@Miauw62 
Yes, however since they have decent armor values for them to be viable exosuits, this would result in a huge armor imbalance when worn together with other armors.

**Comment by Aranclanos** [comment]
>Yes, however since they have decent armor values for them to be viable exosuits, this would result in a huge armor imbalance when worn together with other armors.

haha holy shit some people

**Comment by Anonus** [comment]
These, however, are not actually cloaks, but capes, as they have no hood.

**Comment by TechnoAlchemisto** [comment]
@aranclanos Helo

@Anonus 
Not all cloaks have hoods, a cloak without a hood is not a cape. However, I think that cloaks would we way cooler if they did in fact have hoods, so i'll look into that! I'm still figuring out coding so forgive me.

**Comment by MrStonedOne** [comment]
Please make a changelog,

You can add it to the PR's body rather then as a file now, see here: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157

**Comment by YakumoChen** [comment]
DOESN'T WORK STILL WRONG SLOT

**Comment by TechnoAlchemisto** [comment]
we were so wrong

**Pull request merged**